Documentation/cmdref/cilium-bugtool.md

```diff
@@ -24,7 +24,7 @@ cilium-bugtool [OPTIONS] [flags]
 	NAME                          READY     STATUS    RESTARTS   AGE
 	cilium-kg8lv                  1/1       Running   0          13m
 	[...]
-	$ kubectl -n kube-system exec cilium-kg8lv cilium-bugtool
+	$ kubectl -n kube-system exec cilium-kg8lv -- cilium-bugtool
```

Comment:
This file is autogenerated from https://github.com/cilium/cilium/blob/d88d5f95f3ff0c19fd926abf01ad3f4ee449bf3a/bugtool/cmd/root.go#L49

so you need to change this line as well.

Reply:
Thanks, I've update the PR. Please take a look.

Reply:
@aanm I noticed that ``helpers.bash`` have same issue.
https://github.com/cilium/cilium/blob/master/tests/helpers.bash#L221 
But this PR is related to document, should I send another pull request?

Reply:
@JieJhih no, you can also change it here. It's fine